primitives/std/with_std.rs

```diff
@@ -37,6 +37,7 @@ pub use std::sync;
 pub use std::result;
 pub use std::slice;
 pub use std::str;
+pub use core::time;
```

Comment:
```suggestion
pub use std::time;
```

Reply:
No that was by intention to only include duration.